src/NBitcoin/BIP32/BitcoinExtKey.cs

```diff
@@ -67,7 +67,7 @@ public ExtKey ExtKey
                 if(this._Key == null)
                 {
                     this._Key = new ExtKey();
-                    this._Key.ReadWrite(this.vchData);
+                    this._Key.ReadWrite(this.vchData, network: this.Network);
```

Comment:
After the change we made with DefaultConsensusFactory we don't need network in this code